app/controllers/rounds_controller.rb

```diff
@@ -1,13 +1,22 @@
 class RoundsController < ApplicationController
   before_action :require_login
   before_action :set_round, only: [:show]
-  before_action :set_team, only: [:create]
+  before_action :set_team, only: [:index, :create]
+
+  def index
+    @rounds =
+      if @team.present?
```

Comment:
IMO, this makes sense as I don't see any reason to prevent (on the controllers/routes side, at least) accessing the rounds that exist. One can think on use cases to be able to access this action without `@team` being set, either for admin roles or for "gamification purposes".
And we need not to implement those use cases, of course, but assuming here that another resource's record needs to be defined in this scope is, for me, a shot in the foot.